addons/product_expiry/models/production_lot.py

```diff
@@ -105,21 +105,19 @@ def _alert_date_exceeded(self):
     def _update_date_values(self, new_date):
         if new_date:
             time_delta = new_date - (self.expiration_date or fields.Datetime.now())
-            vals = self._get_date_values(time_delta)
+            vals = self._get_date_values(time_delta, new_date)
             vals['expiration_date'] = new_date
             self.write(vals)
 
-    def _get_date_values(self, time_delta):
+    def _get_date_values(self, time_delta, new_date):
```

Comment:
As you add a new parameter, we should give a default value to be sure we do not break flow of someone who call  it somewhere else 
```suggestion
    def _get_date_values(self, time_delta, new_date=False):
```

Reply:
Yes, indeed!
Thanks for the review